Makefile

```diff
@@ -27,57 +30,68 @@ clean: ## Clean build directory
 	cd $(DOCS_DIR) && rm -rf $(BUILDDIR)/*
 
 .PHONY: build
-build:  ## Set up training: Install requirements
+build:		## Set up training: Install requirements
```

Comment:
Please revert replacement of spaces with tabs for all comment indents.

Reply:
Makefile targets list their dependencies after the colon, without some spacing it might look confusing, so I typically use two tabs to separate the help comment from the rest of the target definition. I will remove the tabs.

Reply:
Thank you. Two spaces is good enough, is familiar to Python inline comments after a statement, and what we've already done with Trainings.

Reply:
This was fixed in a couple of commits 042ad3327316f4a3a7932eb489da4ebf515a5e6b and 024c8fee982632e0998dce17d70a91bf50d53e2b